### ISSUE TITLE
Function/argument name overlays function/argument type in IntelliSense

### ISSUE THREAD

**Issue by adiabatic** [microsoft/vscode-go#2266]
When I write some functions, their names overwrite their types.

A code snippet and screenshot would explain this much better. Here's some code that I have that works just fine in the Playground:

```package main

import (
	"fmt"
	"net/http"
)

type User struct {
	// elided
}

func (mw *User) ApplyFn(next http.HandlerFunc) http.HandlerFunc {
	return http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
		cookie, err := r.Cookie("remember_token")
		if err != nil {
			// the comment on line 16
			return
		}
		_ = cookie
	})
}

func main() {
	fmt.Println("Hello, playground")
}
``` 

Have a look at the comment on line 16 that says its's the comment on line 16. If I delete that comment and type `next(`, I get an IntelliSense tooltip that says "nextlerFunc" (I'm guessing it's "next" laid over "HandlerFunc"). It looks like this:

<img width="943" alt="Visual Studio Code showing “nextler func” in an IntelliSense tooltip" src="https://user-images.githubusercontent.com/101246/51508467-268e0c00-1daa-11e9-9d65-2e327a8faf70.png">

This is more than a little confusing and I've occasionally had variables where being able to see the full type would be useful. Could this be looked into?

**Comment by ramya-rao-a** [comment]
That is indeed weird, but I dont see it.

What you are seeing is the signature help i.e the tooltip that appears when you are typing parameters for a function inside the `()`. But I dont see it.

Can you share your Go related settings?

Can you also retry this after disabling all other extensions? Just to ensure that some other extension is not interfering here?

**Comment by adiabatic** [comment]
I disabled all other extensions and reloaded Visual Studio Code. It still happens.

I'd love to share my Go-related settings, but it seems I don't have any. I went into `Preferences: Open Settings (JSON)` on my laptop and there's nothing there in my laptop. I went to `~/Library/Application Support/Code/User/` and there's no `preferences.json` in there. On my desktop, all my user-overridden settings start with `html.`, `javascript.`, `typescript.`, `workbench.`, or `editor.`.

Sorry I couldn't be more helpful.
